Agents/ForecastingAgent/docker-compose-test_dockerised.yml

```diff
@@ -0,0 +1,48 @@
+version: '3.8'
+
+services:
+  forecasting_agent:
+    image:  ghcr.io/cambridge-cares/forecasting_agent:2.0.0
+    build:
+      context: .
+      target: prod
+    env_file:
+      # Specify environment variables for testing
+      - agent_test.env
+    environment:
+      - OVERWRITE_FORECAST=true
```

Comment:
Is it possible to make `OVERWRITE_FORECAST` something that can be queried from the triple store so it can be decided per derivation? It seems a massive resource requirement to have two agents spun up for both cases - considering each docker container is at least 6.5 GB!

Reply:
Of course, one could do that; however, as it has been decided that overwriting forecasts shall be the default behaviour anyway (as not doing so partially contradicts the DIF), I do not see the point of doing so. We should rather make not overwriting less prominent. Ok?

Reply:
Same as the last comment, I would recommend documenting this somewhere so others know this is only kept in docker-compose for testing purposes.